homeassistant/components/backup/manager.py

```diff
@@ -652,6 +670,7 @@ async def async_create_backup(
             include_homeassistant=include_homeassistant,
             name=name,
             password=password,
+            raise_task_error=True,
```

Comment:
I think we should add a comment explaining why we want to raise here, but not in initiate

Reply:
We can do that later when we've decided if we want to rename the parameter or not.